File: Wizlinx_android/ForSeeSolution/app/src/main/java/telegram/free/forseesolution/RestClient.java
package telegram.free.forseesolution;

import com.google.android.gms.common.api.ApiException;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RestClient {
    private static final String BASE_URL = "https://shoppingtrendonline.com/";
    private static Retrofit retrofit = null;

    public static Retrofit getApiService() {
        if (retrofit == null) {
            retrofit = new Retrofit
                    .Builder()
                    .baseUrl(BASE_URL)
                    .addConverterFactory(GsonConverterFactory.create())
                    .build();
        }
        return retrofit;
    }
}

File: MyApplicationKt/app/src/main/java/telegram/free/myapplicationkt/Utils/RestClient.kt
package telegram.free.myapplicationkt.Utils

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory
import telegram.free.myapplicationkt.ApiServices

object RestClient {
    lateinit var retrofit:Retrofit


        val Url="https://jsonplaceholder.typicode.com"
         fun getRetrofit():ApiServices {
             return Retrofit.Builder()
                .addConverterFactory(
                    GsonConverterFactory.create())
                .baseUrl(Url)
                .build().create(ApiServices::class.java)


    }

}